jetty-server/src/main/java/org/eclipse/jetty/server/PushBuilderImpl.java

```diff
@@ -68,7 +68,7 @@ public PushBuilderImpl(Request request, HttpFields fields, String method, String
         _sessionId = sessionId;
         _fields.add(JETTY_PUSH);
         if (LOG.isDebugEnabled())
-            LOG.debug("PushBuilder({} {}?{} s={} c={})", _method, _request.getRequestURI(), _queryString, _sessionId);
+            LOG.debug("PushBuilder({} {}?{} s={})", _method, _request.getRequestURI(), _queryString, _sessionId);
```

Comment:
I got rid of `c={}`, as I have no idea what `c` could have been.

Reply:
Looks like the constructor used to have a boolean passed in, but it doesn't exist any more. Good catch.